api/models.py

```diff
@@ -91,6 +91,7 @@ def create_superuser(self, email, password, **extra_fields):
 
 class User(AbstractUser):
 
+    LANGUAGE_CHOICES = (("de", "Deutsch"), ("en", "Englisch"))
```

Comment:
```suggestion
    LANGUAGE_CHOICES = (("de", "Deutsch"), ("en", "English"))
```

Die Namen der Sprachen sollten auch in der jeweiligen Sprache sein :)